pkgs/top-level/all-packages.nix

```diff
@@ -2011,11 +2011,7 @@ in
 
   mongodb-compass = callPackage ../tools/misc/mongodb-compass { };
 
-  mongodb-tools = callPackage ../tools/misc/mongodb-tools {
-    # XXX: this is failing with Go 1.12. Error is related to cgo, an
-    # update to this package might fix it.
-    buildGoPackage = buildGo111Package;
-  };
+  mongodb-tools = callPackage ../tools/misc/mongodb-tools { };
```

Comment:
Would `buildGo110Package` be an option?

Reply:
Otherwise if the some of the tools are that unmaintained, we could also think about removing them completely from nixpkgs.

Reply:
Ok you mentioned that in your PR description already.

Reply:
No, because `buildGo110Package` is being removed in https://github.com/NixOS/nixpkgs/pull/68194.